### PR TITLE
[DEVOPS-118] Windows installer: escalate to admin exec level

### PR DESCRIPTION
**Pull request by jmitchell**
Installer currently fails to run for non-admin users without explicitly running it as an admin. To avoid the error we should preemptively request admin permission.

Manually verified using a local build on Windows Server 2012 r2.

### CODE REVIEW

**Comment by domenkozar** [comment]
@fersel can you test this installer with the update system?

File: installers/WindowsInstaller.hs [hunk 1]
```diff
@@ -107,7 +107,7 @@ writeInstallerNSIS fullVersion = do
     injectGlobalLiteral $ "VIAddVersionKey \"ProductVersion\" " <> fullVersion
     injectGlobalLiteral "Unicode true"
     installDir "$PROGRAMFILES64\\Daedalus"   -- The default installation directory
-    requestExecutionLevel Highest
+    requestExecutionLevel Admin
```
Comment (deepfire) [inline installers/WindowsInstaller.hs]: Context for this change:

- http://stackoverflow.com/a/12652091

**Comment by deepfire** [comment]
Quoting http://stackoverflow.com/a/12652091:

> `highestAvailable` will elevate if the current user is an administrator. Which is consistent with what you have observed. When a standard user runs the process, no UAC dialog is shown and the process runs with the standard token. When an admin user executes, the UAC consent dialog is shown and the process will then run elevated.
>
> If your program requires admin rights to function then you need to use `requireAdministrator`. When a standard user starts such a process, the over-the-shoulder UAC dialog is shown. That gives the user an opportunity to ask an admin to supply their credentials.

This makes sense.

File: installers/WindowsInstaller.hs (hunk 1 above)
Comment (jmitchell) [inline installers/WindowsInstaller.hs]: Additional context from [NSIS docs](http://nsis.sourceforge.net/Docs/Chapter4.html#arequestexecutionlevel)

**Pull request merged**